src/biotite/interface/rdkit/mol.py

```diff
@@ -352,22 +398,40 @@ def from_mol(mol, conformer_id=None, add_hydrogen=None):
             extra_annotations[annot_name][_atom_idx] = rdkit_atom.GetProp(annot_name)
 
     for annot_name, array in extra_annotations.items():
-        atoms.set_annotation(annot_name, array.astype(str))
+        # ... handle special case of implicit hydrogen atom flags
+        if annot_name == "isImplicit":
+            atoms.set_annotation(annot_name, array.astype(bool))
+            continue
```

Comment:
Could you elaborate on this, i.e. is there an such an Atom property?

Reply:
Now I also encountered it :+1:. I expanded the comment a bit, explaining that `AddHs()` is the source.